apiserver/admin.go

```diff
@@ -168,6 +169,14 @@ func (a *admin) login(ctx context.Context, req params.LoginRequest, loginVersion
 		a.apiObserver, auditRecorder, auditConfig.CaptureAPIArgs,
 	)
 	a.root.rpcConn.ServeRoot(apiRoot, recorderFactory, serverError)
+
+	// Security Event Logging: This log statement is required to comply with Canonical's SSDLC Security Event Logging policy.
+	// For details, see:
+	// https://library.canonical.com/corporate-policies/information-security-policies/ssdlc/ssdlc---security-event-logging
```

Comment:
This isn't publicly available. I'm hesitant to put links directly into open source code that external contributors can't visit.

I think we could either have our own doc, or summaries the intent from the doc into the comments.

Reply:
Good point; for some reason, I thought this was publicly available. I'll drop the link.